### PR TITLE
[new] add jQRangeSlider@5.7.1 with git auto-update, close #5059

### LINKED ISSUE #5059: [Request] jQRangeSlider

### PR DESCRIPTION
**Pull request by Amomo**
Hi @maruilian11 , I add a new lib with git auto-update from the request #5059 
This lib is lack of minimized, so I attach on minified files.
Would you mind checking this PR for me? Thank you~ :-)

### CODE REVIEW

**Comment by maruilian11** [comment]
@Amomo i think we don't need .gitignore under css/icons-classic.
this case i am confused about what <code>.js, css, png</code> should be included.
@Piicksarn could u give me some advice? thx.

**Comment by PeterDaveHello** [comment]
I don't think `css/**/*` will match `.gitignore`.

**Comment by maruilian11** [comment]
ok. i see.

**Comment by maruilian11** [comment]
@LeaYeh i can't decide the files should be added. can u give me some advice? thank u

**Comment by Piicksarn** [comment]
In my opinion, i will make file map like this:
"jQRangeSlider*.js",
"css/**/*"

**Comment by PeterDaveHello** [comment]
Any updates?

**Comment by Amomo** [comment]
@Piicksarn @maruilian11 Thanks for your reply!
Sorry for the inconvenience, the repo of jQRangeSlider is below.
https://github.com/ghusse/jQRangeSlider
Because files in the repo are not only begin with `jQRangeSlider`, but also contain some files called `jQEditRangeSlider`,` jQDateRangeSlider`, even` jQRuler`. 
That's why I use `"jQ*.js"`. Is it okay?
Hope you could give me some feedback. Thank you very much! :-D

**Comment by maruilian11** [comment]
@Amomo i see~
@PeterDaveHello i think it's fine.

**Comment by PeterDaveHello** [comment]
Need to rebase with latest master branch first.

**Comment by Amomo** [comment]
This branch has matched the latest master branch!

File: ajax/libs/jQRangeSlider/5.7.1/css/icons-classic/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+/*.db
```
Comment (PeterDaveHello) [inline ajax/libs/jQRangeSlider/5.7.1/css/icons-classic/.gitignore]: What's the reason why to add .gitignore here you guys?

**Comment by Amomo** [comment]
Oh, I forgot to pick out this unnecessary file. Sorry.
It has been modified.

**Comment by maruilian11** [comment]
@Amomo i think it's ok, can u rebase it and tag me? thank u.

**Comment by Amomo** [comment]
@maruilian11 I have rebased it. Thank you~ :-D

**Comment by maruilian11** [comment]
@PeterDaveHello i think it's ok.

File: ajax/libs/jQRangeSlider/package.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      "url": "https://github.com/ghusse/jQRangeSlider"
+    }
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/ghusse/jQRangeSlider.git",
+    "basePath": "",
+    "files": [
+      "jQ*.js",
+      "css/**/*"
```
Comment (PeterDaveHello) [inline ajax/libs/jQRangeSlider/package.json]: We don't need this:
https://github.com/ghusse/jQRangeSlider/blob/v4.2.9/css/icons/.gitignore

**Comment by maruilian11** [comment]
@PeterDaveHello but u said "I don't think <code>css/***/**</code> will match <code>.gitignore.</code>"

**Comment by PeterDaveHello** [comment]
Oh yeah, you are correct.

**Comment by PeterDaveHello** [comment]
Thanks.

**Pull request merged**